design/general-progress-monitoring.md

```diff
@@ -157,15 +172,30 @@ any uploads still in progress should be aborted.
 The "WaitingForPluginOperations" phase signifies that the main part of the backup/restore, including
 snapshotting has completed successfully and uploading and any other asynchronous BIA/RIA plugin
 operations are continuing.  In the event of an error during this phase, the phase will change to
-WaitingForPluginOperationsPartiallyFailed.  On success, the phase changes to Completed.  Backups
-cannot be restored from when they are in the WaitingForPluginOperations state.
+WaitingForPluginOperationsPartiallyFailed.  On success, the phase changes to
+"FinalizingAfterPluginOperations" for backups and "Completed" for restores.  Backups cannot be
+restored from when they are in the WaitingForPluginOperations state.
 
 ### WaitingForPluginOperationsPartiallyFailed (new)
 The "WaitingForPluginOperationsPartiallyFailed" phase signifies that the main part of the
 backup/restore, including snapshotting has completed, but there were partial failures either during
 the main part or during any async operations, including snapshot uploads.  Backups cannot be
 restored from when they are in the WaitingForPluginOperationsPartiallyFailed state.
 
+### FinalizingAfterPluginOperations (new)
+The "FinalizingAfterPluginOperations" phase signifies that asynchronous backup operations have all
+completed successfully and Velero is currently backing up any resources indicated by asynchronous
+plugins as items to back up after operations complete. Once this is done, the phase changes to
+Completed.  Backups cannot be restored from when they are in the FinalizingAfterPluginOperations
+state.
+
+### FinalizingAfterPluginOperationsPartiallyFailed (new)
+The "FinalizingAfterPluginOperationsPartiallyFailed" phase signifies that asynchronous backup
```

Comment:
It is a little confusing here, this description has no difference with the one for `FinalizingAfterPluginOperations`, can we add some words to indicate that this state is derived from `WaitingForPluginOperationsPartiallyFailed`?

Reply:
@Lyndon-Li Oh, sorry. Yeah, I messed this block up a bit with the copy-and-paste. Will update now.

Reply:
@Lyndon-Li fixed.